massgov/mayflower-patternlab
# DP-8942: Add sections to elected official page

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

This PR adds missing content sections to the JSON of elected organization pages.

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8942)

## Steps to Test


1. Look at http://localhost:3000/?p=pages-organization-elected-official
2. Confirm the sections match the spec here: https://docs.google.com/document/d/1JnC_IPJEjyqxqtICGPoITkzTxdfjK_tgmuJzNCKE5tE/edit  


#### Impacted Areas in Application


*  @pages/organization-elected-official.json
